app/assets/templates/balances/balance.jst.jade

```diff
@@ -4,8 +4,24 @@
     .balance-box__date
       span Last Change:&nbsp;
       = moment(locals.created_at).format('LL')
-      if locals.transferLink && +locals.balance < 0
-        | &nbsp;
-        a(href=locals.transferLink,data-navigate) Send transfer
+      if locals.transferLink && (+locals.balance < 0 || locals.pending_balance < 0)
```

Comment:
What's the `+` for?

Reply:
Redundant leftover (`+` converts a variable to a number).